rules/busybox.bzl

```diff
@@ -118,6 +118,12 @@ def _make_resources_flag(
         ],
     )
 
+def _disable_warnings(ctx, args):
+    if (ctx.fragments.android.persistent_android_resource_processor or
```

Comment:
I changed these fragment flags to `ctx.fragments.android.persistent_busybox_tools` and `ctx.fragments.android.persistent_multiplex_busybox_tools` internally and tests are passing. Those flags also seem more appropriate than the resource_processor ones; what do you think?

Unfortunately the `fragments.android` data structure can't accept `Void` expansion flags such as persistent_android_resource_processor, and changing the return type of that flag handler to `boolean` causes some other kind of sanity check error. I think in Bazel/Blaze, expansion flags must return Void, so there's somewhat of an impedance mismatch.